### PR TITLE
[INLONG-2017] Add more guide documents for Pulsar

### PR DESCRIPTION
**Pull request by dockerzhang**
Fixes https://github.com/apache/incubator-inlong/issues/2017

where *XYZ* should be replaced by the actual issue number.

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by healchow — APPROVED** [review]

File: docs/user_guide/user_manual.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ Click [Create], there are two steps to fill in data access information: Group in
 
 Access requirements require users to choose message middleware: high throughput (TUBE) or High reliability (PULSAR):
```
Comment (gosonzhang) [inline docs/user_guide/user_manual.md:27]: TubeMQ

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user_guide/user_manual.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ sidebar_position: 1
 
 接入要求需要用户进行选择消息中间件：高吞吐（TUBE）或者高可靠（PULSAR）：
```
Comment (gosonzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/user_guide/user_manual.md:26]: TubeMQ

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/quick_start.md [hunk 3]
```diff
@@ -38,21 +38,35 @@ spring.profiles.active=dev
    spring.datasource.password=DB_PASSWD
    ```
 
-2) 修改 Tube 和 ZooKeeper 集群的连接信息，其中 `cluster.zk.root` 建议使用默认值：
+2) 配置消息队列服务，可以使用InLong TubeMQ 或者 Apache Pulsar：
 
+- 若使用TubeMQ，配置TubeMQ 集群信息
    ```properties
    # Tube 集群的 Manager 地址，用来创建 Topic
```
Comment (gosonzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/quick_start.md:45]: TubeMQ

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/quick_start.md [hunk 4]
```diff
@@ -38,21 +38,35 @@ spring.profiles.active=dev
...
    cluster.tube.manager=http://127.0.0.1:8081
    # 用来管理 Tube 的 Broker
    cluster.tube.master=127.0.0.1:8000,127.0.0.1:8010
```
Comment (gosonzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/quick_start.md:48]: tubemq

**Review by gosonzhang — CHANGES_REQUESTED** [review]

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**